Makefile

```diff
@@ -24,7 +24,15 @@ verify-podnetworkconnectivitychecks:
 	$(MAKE) -C pkg/operator/connectivitycheckcontroller verify
 
 test-e2e-encryption: GO_TEST_PACKAGES :=./test/e2e-encryption/...
+test-e2e-encryption: GO_TEST_FLAGS += -v
```

Comment:
let's not change flags for `test-e2e-encryption` in this PR.

Reply:
can you remove changes to `test-e2e-encryption`? this part will be updated in https://github.com/openshift/library-go/pull/1470